WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2305,7 +2305,9 @@ private GutenbergPropsBuilder getGutenbergPropsBuilder() {
         String languageString = LocaleManager.getLanguage(EditPostActivity.this);
         String wpcomLocaleSlug = languageString.replace("_", "-").toLowerCase(Locale.ENGLISH);
 
-        // If this.mIsXPostsCapable has not been set, default to allowing xPosts
+        // this.mIsXPostsCapable may return true for non-WP.com sites, but the app only supports xPosts for P2-based
+        // WP.com sites.
```

Comment:
```suggestion
        // WP.com sites so, gate with `isUsingWpComRestApi()`
```

Bit of a nit but, WDYT about the above suggestion instead of the `If the site is using the WP.com API`? I find the separate sentece for the API gate slightly confusing, and the "If ..." is a bit verbose since it can be seen in the code that it's a conditional.

Reply:
Sounds like a good iteration to me! I've removed `If the site is using the WP.com API` in https://github.com/wordpress-mobile/WordPress-Android/pull/16547/commits/5f087bff30e00809b36033e36e15bf4b72912c78 and will go ahead to commit your suggestion.